contracts/contracts/keeper/Keeper.sol

```diff
@@ -0,0 +1,194 @@
+pragma solidity 0.5.11;
+
+pragma experimental ABIEncoderV2;
+
+import "hardhat/console.sol";
+
+/**
+ * @title OUSD Keeper Contract
+ * @notice The Keeper is used to perform maintenance on vaults
+ * @author Origin Protocol Inc
+ */
+
+import { IKeeper } from "../interfaces/IKeeper.sol";
+import "../vault/VaultCore.sol";
+import "../token/OUSD.sol";
+import "../governance/Governable.sol";
+
+contract Keeper is IKeeper, Governable {
```

Comment:
We might want to call this contract something like "KeeperConnector", since otherwise it could get confused with with the Keeper network itself.

Reply:
Done - I agree